### PR TITLE
Add city and theme on viewpoint model

### PR DESCRIPTION
**Pull request by AL9000**
+ migrate existing data on migration

### CODE REVIEW

**Comment by AL9000** [comment]
I still need to figure out why those two tests are not OK :
test_viewpoint_create_with_related_docs
and
test_viewpoint_create_with_related_docs

When I just set `format="multipart"` on the first one it's OK, but still need to know why :)

**Comment by AL9000** [comment]
to myself: @AL9000 Make more tests, see coverage ;)

File: terra_opp/migrations/0003_auto_20200630_1315.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            name='city',
+            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, related_name='viewpoints', to='terra_opp.City'),
+        ),
+        migrations.AddField(
+            model_name='viewpoint',
+            name='themes',
+            field=models.ManyToManyField(related_name='viewpoints', to='terra_opp.Theme'),
+        ),
+        migrations.RunPython(set_city_and_themes_from_properties, reverse_code=migrations.RunPython.noop),
+    ]
```
Comment (jrmi) [inline terra_opp/migrations/0003_auto_20200630_1315.py:63]: Remove old field

Comment (AL9000) [reply terra_opp/migrations/0003_auto_20200630_1315.py:68]: Done!

File: terra_opp/views.py [hunk 2]
```diff
@@ -135,6 +141,22 @@ def filters(self, request, *args, **kwargs):
...
+        )
+        filter_values['themes'] = Theme.objects.exclude(label__isnull=True).exclude(
+            label__exact='',
+        ).distinct().order_by(
+            'label',
+        ).values_list(
+            'label',
+            flat=True,
+        )
+
```
[the remark this answers is not in the document]
Comment (submarcos) [reply terra_opp/views.py:159]: :+1:

**Review by submarcos — APPROVED** [review]

**Pull request merged**